app/commands_app/commands/get_pods.py

```diff
@@ -10,10 +11,11 @@ def parse_args(sub_parser):
 def get_pods(arg):
     print(get_aladdin_demo_pods())
 
+
 def get_aladdin_demo_pods():
     config.load_incluster_config()
     v1 = client.CoreV1Api()
     res = v1.list_namespaced_pod(
         namespace=os.environ["NAMESPACE"],
-        label_selector=f"project={os.environ['PROJECT_NAME']}")
+        label_selector="project=" + os.environ['PROJECT_NAME'])
```

Comment:
Discovered this by accident --- but it looks like our default commands pod (or at least the image we depend on in aladdin-demo) still only has python 2.  We should fix this, but for now, commands can't contain python 3 syntax.